edward/Achrismastree/src/main/java/foo/Foo.java

```diff
@@ -0,0 +1,79 @@
+package foo;
+import java.util.Scanner;
+
+public class Foo {
+    public static void main(String[] args){
+        System.out.println("Enter a positive integer number");
+        Scanner number_receiver = new Scanner(System.in);
+
+        int y = 0;
+        int x = number_receiver.nextInt();
+
+        if(x < 1){
+            System.out.println("!@#*@*&$*@&$*@(!))@$&*(@$&*!&(@%@)$&**(@!");
+            return;
+        }
+        while(0 == 0) {
+            System.out.println("          1          ");
+            System.out.println("         2 2         ");
+            System.out.println("        3   3        ");
+            System.out.println("       4     4       ");
+            System.out.println("      5       5      ");
+            System.out.println("       6     6       ");
+            System.out.println("      7       7      ");
+            System.out.println("     8         8     ");
+            System.out.println("    9           9    ");
+            System.out.println("   0             0   ");
+            System.out.println("    VAVAVAVAVAVAV    ");
+

```

Comment:
When I say loop, I mean to use loop to print the tree, not print multiple trees in a loop. Can you try just using loop to print the above tree ?